### ISSUE TITLE
item span size does not work

### ISSUE THREAD

**Issue by walterFeng** [Codewaves/Sticky-Header-Grid#6]
When first load data,  if there have different spanSize of items in the same group, SpanSizeLookup will does not work.

can you help me to fix the issues, thank you very much!

**Comment by Codewaves** [comment]
Hi,

It is hard to say what is wrong without sample code. Make sure you are calculating span info correctly. Maybe if you describe the problem with more details, I can find where is the problem.

**Comment by walterFeng** [comment]
hi,
In you demo(SampleActivity.java), modify some code about LayoutManager.spanSizeLookup like this:
```
 @Override
public int getSpanSize(int section, int position) {
     if (position > 2) {
        return 2;
      else {
        return 1;
      }
}
```
then, scroll recyclerView , you can get it.
Please forgive me for my poor English ! thank you.

**Comment by Codewaves** [comment]
Thanks for the bug report. Fixed in 0.9.6

**Comment by walterFeng** [comment]
It works for me and looks very nice. thanks a lot!!
